content/en/workflows/build.md

```diff
@@ -3,7 +3,141 @@ title: Build Workflows
 kind: documentation
 disable_toc: false
 further_reading:
-- link: "logs/processing/pipelines"
+- link: "/workflows/actions_catalog"
   tag: "Documentation"
-  text: "Log processing pipelines"
----
\ No newline at end of file
+  text: "Browse the available actions in the Actions Catalog"
+---
+
+You can create workflows or edit existing workflows from the [**Workflows** page][1] in Datadog. The Workflows page lists existing workflows together with each workflow's author and the dates that each workflow was last modified and executed.
+- Hover over a workflow for the options to delete or clone the workflow.
+- Toggle **My workflows** if you want to see only workflows that you created.
+
+## Build a workflow from a blueprint
+
+1. Click the **Blueprints** tab.
+1. If desired, use the search bar to narrow the list of blueprints by name, category, or integration.
+1. Find the blueprint you would like to use, and click on it. The workflow canvas appears.
+1. Click the pencil icon to update the workflow name. If desired, type a new name in the text box and click **Save**.
```

Comment:
This line maybe needs the same edit you made to my other PR? Add the tooltip text (**Edit**) in bold?